src/pages/graphql/payment-methods/payment-services-paypal-vault.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Payment Services Paypal Vault payment method
+description: Learn how to use the GraphQL API mutation for the Payment Services Paypal Vault payment solution.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment Services Paypal Vault payment method
+
+Payment Services Paypal Vault is a payment method where a stored card is used for the transaction.
```

Comment:
```suggestion
The [Payment Services Vault payment method](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/payments-checkout/vaulting.html) accesses credit card credentials that the customer has previously saved—or “vaulted”.
```

Reply:
we have a conflicting suggestion with Shrie's here: https://github.com/AdobeDocs/commerce-webapi/pull/250#discussion_r1410843438